packages/esm-patient-registration-app/src/config-schemas/openmrs-esm-patient-registration-schema.ts

```diff
@@ -27,8 +27,8 @@ export const esmPatientRegistrationSchema = {
       },
     },
     _default: {
-      demographics: { name: 'Basic Info', fields: ['name', 'gender', 'dob', 'id'] },
-      contact: { name: 'Contact Details', fields: ['address', 'phone', 'email'] },
+      demographics: { name: 'Basic Info', fields: ['name', 'gender & dob', 'id'] },
```

Comment:
What if I want only gender? Or only date of birth?

Reply:
The mockup wanted them to be in the same section under `Sex & Birth`. I'll confirm from Ciaran about this and will respond back asap.

Reply:
This is not about the *current* design, but about the *configurability*.

Reply:
Just to be clear: I am not questioning the *current* design, I am questioning why we have *configurability* here - my questions above should only point out that these configuration options make no sense as they don't even allow such basics things to be done.

Reply:
I am sorry, but I don't have much idea about the configurations, so should I keep it to discuss it on the talk on Thursday?

Reply:
Let's just leave it like that. I'll take care of it later.

Reply:
Then I should keep it the way it is or revert back all the config changes?

Reply:
Keep it as you have it now.